app/models/case_log.rb

```diff
@@ -110,6 +110,14 @@ def la
     end
   end
 
+  def location_admin_district
+    if location
+      location.location_admin_district
+    else
+      super
```

Comment:
Do we have an attribute on `CaseLog` for this? Isn't this equivalent to LA? (LA has the code but the code can be translated?)

Reply:
i.e. What is `la` and what is `location_admin_district`?

Reply:
Now this is where I also struggled in the beginning - or maybe I still do? In our conversation people seem to refer to the LA by its name ( e.g. "Westmintser" ), however in codebase, this appears to be the actual CODE ( e.g. "E08000003" ). Not aware of 'translation' between those two. In my previous card I coded to add **location_admin_district** column to the location, to populate on create/update alongside with the **location_code**. 
location_admin_district - this name comes from API output, same named field holds this value.

So to sum up for **location**:

location_code is this "E08000003" ( presumably the same what la is )
location_admin_district is this -  "Westminster"

Reply:
Right so currently we have all of these values in the form definition JSON (since we need to be able to ask people for the location if they don't know the postcode). So we should only need to store LA value *E08000003* and then should be able to get/display LA label by doing:

```ruby
case_log.form.get_question("la").label_from_value("case_log["la"])
```